exampleSite/content/posts/hiding-the-date.md

```diff
@@ -0,0 +1,27 @@
+---
+author: "Dave Kerr"
+date: 2020-06-11
+linktitle: Hiding the Date
+title: Hiding the Date
+weight: 10
+hideDate: true
+---
+
+
+## Introduction
+
+Hiding the date in a page or post is easy! Just set `hideDate` to `true` in your front matter.
+
+As an example, the front-matter for this page is:
```

Comment:
Hugo's site spells `front matter` without the hyphen.

Reply:
Thanks, fixed!